usage/jsgui/src/main/webapp/assets/css/base.css

```diff
@@ -362,24 +360,31 @@ input[type="color"]:focus,.uneditable-input:focus {
 }
 
 .addApplication {
-    border: 1px solid #a1cb8c !important;
-    color: #505050 !important;
-    background: url(../images/addApplication-plus.png) no-repeat !important;
-    padding: 10px 0px 0px 74px !important;
-    width: 298px !important;
-    height: 201px !important;
+    padding-top: 5px;
+    font-size: 145%;
     margin-right: 0;
+    color: white;
+    background-color: #90C858;
+  *background-color: #609040;
+  background-image: -ms-linear-gradient(top, #90C858, #609040);
+  background-image: -webkit-gradient(linear, 0 0, 0 100%, from(#90C858), to(#609040));
+  background-image: -webkit-linear-gradient(top, #90C858, #609040);
+  background-image: -o-linear-gradient(top, #90C858, #609040);
+  background-image: -moz-linear-gradient(top, #90C858, #609040);
+  background-image: linear-gradient(top, #90C858, #609040);
+  background-repeat: repeat-x;
+  border-color: #609040 #609040 #609040;
+  border-color: rgba(0, 0, 0, 0.1) rgba(0, 0, 0, 0.1) rgba(0, 0, 0, 0.25);
```

Comment:
I had to read up a little - am I right that if a browser understands the `rgba` format it will use the second definition of `border-color` and otherwise it will use the first (#609040)?